Lighthouse Camera/UI/Main/AlertHelper.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  AlertHelper.swift
+//  Lighthouse Camera
+//
+//  Created by Punam Dahiya on 1/26/17.
+//  Copyright © 2017 Lighthouse. All rights reserved.
+//
+
+extension UIAlertController {
+  // Customise UIAlertController to handle permission failure alerts
+
+  enum AlertMessageID {
+    case CameraFailed, MicrophoneFailed
+  }
+
+  convenience init(messageID: AlertMessageID) {
+    switch messageID {
+    case .CameraFailed:
```

Comment:
I don't know if there's consensus on this stuff, but AFAICT this is the recommended formatting for switch statements in Swift. E.g. https://github.com/ochococo/Code-Formatting-In-Swift#control-operator-switch